sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -641,7 +641,8 @@ public void captureUserFeedback(final @NotNull UserFeedback userFeedback) {
     final SentryId sentryId = event.getEventId();
 
     final SentryEnvelopeHeader envelopeHeader =
-        new SentryEnvelopeHeader(sentryId, options.getSdkVersion(), traceContext);
+        new SentryEnvelopeHeader(
+            sentryId, options.getExperimental().getSessionReplay().getSdkVersion(), traceContext);
```

Comment:
I'd add a comment here saying that `options.getExperimental().getSessionReplay().getSdkVersion()` by default equals `options.getSdkVersion()` and can be overwritten by hybrid sdks